sdks/python/tox.ini

```diff
@@ -410,8 +410,9 @@ deps =
     tensorflow>=2.12rc1,<2.13
     # Help pip resolve conflict with typing-extensions for old version of TF https://github.com/apache/beam/issues/30852
     pydantic<2.7
-    protobuf==4.25.5
-extras = test,gcp,ml_test
+extras = test,gcp
+commands_pre =
+  pip install -U 'protobuf==4.25.5'
```

Comment:
install_requires will install protobuf with 5.x, which is not supported by tesnsorflow 2.12.x. So we reinstall protobuf with the fix version here.